shared/js/background/redirect.es6.js

```diff
@@ -48,7 +48,7 @@ function handleRequest (requestData) {
 
     let thisTab = tabManager.get(requestData)
```

Comment:
Perhaps we should actually address why _blank is triggering the tab not being loaded into the tabManager as this seems a bigger concern to me.

Reply:
I think the error happens on the `main_frame` request, which further down should create the tab object.

Reply:
Yeah just seen that actually thanks :)